### DISCUSSION TITLE
Change the target branch for creating PR

### DISCUSSION THREAD

**Discussion by Dimmsen** [renovatebot/renovate#20045] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

newest?

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have the use case, that I want the renovate-bot to check dependencies of a repository and then (if changes are needed) create a pull request to a target branch that I will specify. At the moment I can't find anything that does this and default it always uses the "master" as target branch. Reason for this is that any changes first have to be merged into a develop branch before they get merged onto the master.

### What you tried so far.

/

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
use baseBranches. then renovate scans and opens PR's to only those branches

**Reply by Dimmsen** [reply]
Thought I tried that already. But maybe it didn't work because there was no dependency to update. Thank you

**Reply by Dimmsen** [reply]
Ok for some reason this doesn`t work as expected. I am using baseBranches and a different branch than master now in my config.js and it still creates an initial pull request "Configure Renovate" with "master" as target branch. I want the target to be something different?

**Reply by rarkins** [reply]
That's expected. You must have a renovate config in the default branch.
